### PR TITLE
Add doc for cluster diagnose in dashboard

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? (Required)

Add docs for cluster diagnose access, report in dashboard.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- No

### CODE REVIEW

File: dashboard/diagnose/report.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+示例：
+
+![Scheduler Config Change History 报表](/media/dashboard/diagnose/config-change.png)
+
+上面报表显示，leader-schedule-limit 配置参数在报告时间范围内有被修改过：
+
+* 2020-05-22T20:00:00+08:00，即报告的开始时间 leader-schedule-limit 的配置值为 4，这里并不是指该配置被修改了，只是说明在报告时间范围的开始时间其配置值是 4。
+* 2020-05-22T20:07:00+08:00，leader-schedule-limit 的配置值为 8，说明在 2020-05-22T20:07:00+08:00 左右，该配置的值被修改了。
+
+
```
Comment (lilin90) [inline dashboard/diagnose/report.md:305]: 多余空行会导致 CI lint 失败，只能留一个空行。
```suggestion
```

File: dashboard/diagnose/access.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Cluster Diagnose Access 集群诊断访问
+category: how-to
+aliases: ['/docs-cn/dev/how-to/???']
```
Comment (breeswish) [inline dashboard/diagnose/access.md:4]: What alias should we fill?

File: dashboard/diagnose/access.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: Cluster Diagnose Access 集群诊断访问
```
Comment (breeswish) [inline dashboard/diagnose/access.md:2]: Should it be "集群诊断页面"?

File: dashboard/diagnose/access.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+category: how-to
+aliases: ['/docs-cn/dev/how-to/???']
+---
+
+# 集群诊断页面
+
+集群诊断是在指定的时间范围内，对集群可能存在的问题进行诊断，并将诊断结果和一些集群相关的负载监控信息汇总成一个诊断报告。诊断报告是网页形式，通过浏览器保存后可离线浏览和传阅。
+
+> 注意：
```
Comment (breeswish) [inline dashboard/diagnose/access.md:11]: The style should be 

```
> **注意：**
>
> ...
```

File: dashboard/usage.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
+---
```
Comment (breeswish) [inline dashboard/usage.md:1]: the file location is incorrect? @crazycs520

Comment (crazycs520) [reply dashboard/usage.md:1]: great, done.

File: dashboard/dashboard-diagnose-access.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+category: how-to
+---
+
+# 集群诊断页面
+
+集群诊断是在指定的时间范围内，对集群可能存在的问题进行诊断，并将诊断结果和一些集群相关的负载监控信息汇总成一个诊断报告。诊断报告是网页形式，通过浏览器保存后可离线浏览和传阅。
+
+> **注意：**
+>
+> 集群诊断功能依赖于集群中部署有 Prometheus 监控组件，参见 TiUP 或 TiDB Ansible 部署文档了解如何部署监控组件。若集群中没有部署监控组件，生成的诊断报告中将提示生成失败。
```
Comment (breeswish) [inline dashboard/dashboard-diagnose-access.md:12]: 部署文档建议给出链接

File: dashboard/dashboard-diagnose-access.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (breeswish) [inline dashboard/dashboard-diagnose-access.md:1]: 好像需要更新 TOC.md？我看这里引入了新文件

File: dashboard/dashboard-diagnose-report.md [hunk 8]
```diff
@@ -0,0 +1,362 @@
+---
+title: 集群诊断页面
+category: how-to
+---
+
+# 集群诊断页面
+
+本文档主要介绍诊断报告的内容以及查看技巧，访问集群诊断和生成报告请参考 [诊断报告访问文档](/dashboard/diagnose/access.md)
```
Comment (breeswish) [inline dashboard/dashboard-diagnose-report.md:8]: dead link.

**Comment by TomShawn** [comment]
@crazycs520 Please change the image file names according to https://github.com/pingcap/docs-cn/pull/3359.

**Comment by breeswish** [comment]
@crazycs520 Please create a PR to fix the typos listed above

**Pull request merged**